centchain/api.go

```diff
@@ -92,7 +92,8 @@ func (dsa *defaultSubstrateAPI) GetBlock(blockHash types.Hash) (*types.SignedBlo
 }
 
 func (dsa *defaultSubstrateAPI) GetStorage(key types.StorageKey, target interface{}, blockHash types.Hash) error {
-	return dsa.sapi.RPC.State.GetStorage(key, target, blockHash)
+	_, err := dsa.sapi.RPC.State.GetStorage(key, target, blockHash)
```

Comment:
@philipstanislaus  Since the new API returns bool, differentiating between empty or not, I am not sure if the downstream impact of subsequent calls if empty. Any pointers?